### PR TITLE
BF: no need to byte_type an in which later %s-ed, assume sending bytes registers bytes

### PR DESCRIPTION
**Pull request by yarikoptic**
also adjusted bogus shebang to not be real as in other files

### CODE REVIEW

**Comment by yarikoptic** [comment]
feedback on the suggested fixes would be appreciated, @gabrielfalcao

File: tests/unit/test_core.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+# #!/usr/bin/env python
```
Comment (akun) [inline tests/unit/test_core.py]: 2 `#`?

Comment (yarikoptic) [inline tests/unit/test_core.py]: Just inline with other files. IMHO all those should be gone

**Comment by gabrielfalcao** [comment]
thank you @yarikoptic

**Pull request merged**